reactive/calico.py

```diff
@@ -752,3 +756,16 @@ def get_network(cidr):
 def get_networks(cidrs):
     '''Convert a comma-separated list of CIDRs to a list of networks.'''
     return [get_network(cidr) for cidr in cidrs.split(',')]
+
+
+def is_rpf_config_mismatched():
+    # This check should only apply on focal and above
+    release = lsb_release()
+    release_number = float(release['DISTRIB_RELEASE'])
+    if release_number >= 20.04:
```

Comment:
does this charm only run on `ubuntu`?  I know some charms also support operations on centos.  I hadn't thought about it up to this point. 

`charmhelpers.core.host` goes some distance with abstracting the OS version away. 

even if `ubuntu` is the only choice here maybe consider

```python
from charmhelpers.core.host import CompareHostReleases

...
_release = lsb_release()['DISTRIB_CODENAME'].lower()
if CompareHostReleases(_release) <= 'focal'
```